src/logger.ts

```diff
@@ -77,4 +79,4 @@ async function log(level, ...args) {
     });
 }
 
-module.exports = { setup, error, warn, info, debug };
+export default { setup, error, warn, info, debug };
```

Comment:
should be able to avoid this with a `export function error(...` and such

Reply:
Right, actually I did that first, but I prefer this style, as it's easy to grab the module as `logger` (similar to the native `console`), so I think it's fine.